stable/traefik/templates/dashboard-ingress.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  annotations:
+  {{- range $key, $value := .Values.dashboard.ingress.annotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{- end }}
```

Comment:
Testing this, I found that you're really going to have to wrap this in a double-if condition similar to what you have done in #414.